prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -890,9 +894,13 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct rxr_pkt_entry *pkt
  */
 void rxr_pkt_handle_recv_error(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry, int err, int prov_errno)
 {
-	FI_WARN(&rxr_prov, FI_LOG_CQ,
-			"Packet receive error: %s (%d)\n",
-			efa_strerror(prov_errno), prov_errno);
+	/* Avoid excessive warn logging */
+	if (err != FI_EAGAIN) {
+		FI_WARN(&rxr_prov, FI_LOG_CQ,
+				"Packet receive error: %s (%d)\n",
+				efa_strerror(prov_errno), prov_errno);
+	}
```

Comment:
Same as about, for real errors, this function eventually call `rxr_cq_write_rx_error`or `efa_ep_write_error` so this print is not useful.